src/lib/mapping.js

```diff
@@ -70,7 +70,8 @@ function mapTeamList (row) {
     description: row.description,
     path: row.path,
     organizationId: row.org_id,
-    usersCount: parseInt(row.users_count)
+    usersCount: parseInt(row.users_count),
+    total: row.total
```

Comment:
To recap:

Following our discussion I added as metadata: `currentPage, pageSize, totalPages, totalUsersCount`
`totalPages` and `totalUserCount` can be calculated one from the other, they are kind of redundant. `totalPages` can be removed.
Indeed `pageSize` aka `limit` and currentPage are sent by the user.

I see the following possible aspects:
- Currently we have no default and no max limits. If these are added the metadata sent by user may be different than the returned one, I would keep the `pageSize` aka `limit` in metadata,
- On the other side more metadata means more complexity on server. `totalPages` could be removed